styles/editMember.css

```diff
@@ -0,0 +1,209 @@
+#editMemberform {
+
+    margin-left: 145px;
+    margin-top: 48px;
+      
+}
+
+body{
+    background-color: black;
```

Comment:
You should look at Kratti's PR #10, you still need to import everything from the `template.php` file over so that you'd get the navbar and other global styling